package.json

```diff
@@ -20,8 +20,11 @@
     "prerelease": "pnpm run build",
     "release": "standard-version --skip.changelog",
     "release:dry-run": "standard-version --skip.changelog --dry-run",
-    "release:rc": "pnpm release -- --prerelease rc",
-    "release:rc:dry-run": "pnpm release:rc -- --dry-run",
+    "release:rc": "standard-version --skip.changelog --prerelease rc",
```

Comment:
I would recommend https://github.com/absolute-version/commit-and-tag-version since it is a replacement version. Or consider Release Me! I also use it for my own library https://github.com/needle-di/needle-di 🚀

Reply:
Yep I need to have a look at it! To be honest just used `standard-version` because I was already familiarised with that, I dont even use it for profissional work anymore ;).

Reply:
@dirkluijk I did some small changes related with the lenient changes released recently, mainly to improve / fix types, feel free to check 👍 , thank you! And let me know ofc if you have anything against or to improve, cheers!